### PR TITLE
Slack guide page

### LINKED ISSUE #1414: Add "Intro to Slack" video on join page
# Feature request
Put https://youtu.be/m2JuAa6-ors on our website and link to it on the join page.

### Proposed solution

- Create a new page to house a Youtube video.
- The page should be titled: "Slack Guide" navigable via the `/slack_guide` path.
- It should use the Hero and Content components. [Component documentation here](https://storybook.operationcode.org).
- On the join page, at the end of the first paragraph put `Unfamiliar with Slack? Learn how to use it!` at the end. “Learn how to use it!” should be the content of a link.

### PR DESCRIPTION
**Pull request by calincionca35**
# Description of changes

Creates Slack Guide page with requested video
# Issue Resolved

Fixes #1414

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,7 +129,7 @@
     "express": "^4.17.1",
     "faker": "^5.2.0",
     "file-loader": "^6.2.0",
-    "husky": "^4.3.8",
+    "husky": "^5.1.3",
```
Comment (kylemh) [inline package.json:132]: Why did this get updated?

Comment (kylemh) [reply package.json:132]: The lock file should also have no changes

File: pages/slack_guide.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+import Content from 'components/Content/Content';
+import styles from 'styles/slack_guide.module.css';
+
+const pageTitle = "Slack Guide";
```
Comment (kylemh) [inline pages/slack_guide.js:6]: You're hitting a lint error because " was used instead of '

Comment (calincionca35) [reply pages/slack_guide.js:6]: Thank you for the review, I will make these corrections.

File: package.json (hunk 1 above)
Comment (calincionca35) [reply package.json:132]: I made the update locally, but I will revert to the 4.3.8 version.

**Comment by kylemh** [comment]
#### 📝Changed routes:


| [**/slack_guide**](https://operation-code-fsa72w13o-operation-code.vercel.app/slack_guide) |
|:-:|
|<a href="https://operation-code-fsa72w13o-operation-code.vercel.app/slack_guide">Screenshot of /slack_guide</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/9sBRzkAs7NYs1gwux02DhY0i3jc_.png">(view full size)</a> |







Commit 00e83ad4e71e28e11036f9061607d2315336b3fc (https://operation-code-fsa72w13o-operation-code.vercel.app).

File: pages/slack_guide.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          <iframe
+            title="Slack Guide"
+            src="https://youtube.com/embed/m2JuAa6-ors"
+            frameBorder="0"
+            allowFullScreen
+            width="755"
+            height="425"
+          />,
+        ]}
+      />
```
Comment (kylemh) [inline pages/slack_guide.js:26]:
```suggestion
      <HeroBanner className={styles.hero} title={pageTitle}>
        <iframe
          title="Slack Guide"
          src="https://youtube.com/embed/m2JuAa6-ors"
          frameBorder="0"
          allowFullScreen
          width="755"
          height="425"
        />
      </HeroBanner>
```

try this for me, would you?

File: components/Forms/RegistrationForm/RegistrationForm.js [hunk 4]
```diff
@@ -12,7 +12,9 @@ import Button from 'components/Buttons/Button/Button';
 import Form from 'components/Form/Form';
 import Input from 'components/Form/Input/Input';
 import Alert from 'components/Alert/Alert';
+import Link from 'next/link';
 import styles from './RegistrationForm.module.css';
+// import OutboundLink from 'components/OutboundLink/OutboundLink';
```
Comment (kylemh) [inline components/Forms/RegistrationForm/RegistrationForm.js:17]: can you delete this line?

File: pages/slack_guide.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+import Content from 'components/Content/Content';
```
Comment (kylemh) [inline pages/slack_guide.js:3]: can u delete this line since it's unused?

**Review by kylemh — COMMENTED** [review]
just these deletes and make sure there's no other lint errors, and we're good to ship!

**Review by kylemh — APPROVED** [review]

**Pull request merged**